pkgs/applications/networking/browsers/firefox/common.nix

```
@@ -217,6 +214,7 @@ buildStdenv.mkDerivation ({
     wrapGAppsHook
   ]
   ++ lib.optionals pgoSupport [ xvfb-run ]
+  ++ lib.optionals crashreporterSupport [ dump_syms ]
```

```suggestion
  ++ lib.optionals crashreporterSupport [ dump_syms ]
  ++ lib.optionals pgoSupport [ xvfb-run ]
```

Alphabetical ordering